plugin.xml

```diff
@@ -50,6 +50,7 @@
             <meta-data android:name="VERSION" android:value="1" />
             <meta-data android:name="QUERY_LOG" android:value="false" />
             <meta-data android:name="DOMAIN_PACKAGE_NAME" android:value="com.spoon.backgroundfileupload" />
+            <!-- separate process -->
```

Comment:
@sc-nick les aidl sont supportés ? https://github.com/AlexDisler/cordova-plugin-inapppurchase/blob/master/src/android/billing/IInAppBillingService.aidl

Mais peut-être que des threads nous suffiront ?
> A commonly used pattern for a service implementation is to create and run a new Thread in the service to perform the processing in the background and then to terminate the service once it has finished the processing.
https://www.vogella.com/tutorials/AndroidServices/article.html?utm_source=dlvr.it&utm_medium=feed

Ou sinon, on peut attendre voir qu'est-ce que ça donne sans threads ou process séparé..